app/interactors/tasks/prepare_params.rb

```diff
@@ -0,0 +1,19 @@
+module Tasks
+  class PrepareParams
```

Comment:
Shouldn't it be in Tasks::Create::PrepareParams? Isn't this params specific for task creation?

Reply:
i think that prepare params - stage which needed be before creating stage

Reply:
The order of the call is determined by the organizer, the nesting of modules allows us to understand that this interactor is specific to the creation action.